FourthTask/figures/types/types.go

```diff
@@ -0,0 +1,58 @@
+//	import $GOPATH/src/github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types
+//	Package contains structures descriptions and constructors, interface definition
+
+package types
+
+import (
+	"errors"
+	"math"
+)
+
+var (
+	ErrCircleNegative = errors.New(" circle can't has negative radius")
+	ErrSquareNegative = errors.New(" square can't has negative side")
+	overflowError     = errors.New(" result will be bigger, that we can coutn. sorry...")
+)
+
+type Figure interface {
+	Area() (float64, error)
+	Perimetr() (float64, error)
```

Comment:
Do we really need `error` result in these functions ?  You always return `nil` as a second result value.